intercom-java/src/main/java/io/intercom/api/User.java

```diff
@@ -91,7 +91,7 @@ static UserUpdate buildFrom(User user) {
             final UserUpdate userUpdate = new UserUpdate();
             userUpdate.userId = user.getUserId();
             userUpdate.email = user.getEmail();
-            // userUpdate.id = user.id; // bug: id is treated as custom attribute
+            userUpdate.id = user.getId();
```

Comment:
@dehora is this still an issue? Test seems to produce the correct result using `id` here